### DISCUSSION TITLE
Problem with loading text

### DISCUSSION THREAD

**Discussion by liamreckley** [mdbergmann/Eloquent#75] [General]
For some reason on Mac, the software keeps crashing. Further after importing modules, the text does not actually show up. So nothing imported actually works. Any solutions are appreciated. Thank you!

**Comment by mdbergmann** [comment]
Hello.

Assumingly you are using the latest version (v 2.6.7)?

**Reply by liamreckley** [reply]
Correct, the most updated version.

**Reply by mdbergmann** [reply]
Are there any crash logs under ~/Library/Logs/CrashReporter ?
